src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OperatorsShouldHaveSymmetricalOverloads.cs

```diff
@@ -16,7 +18,10 @@ public abstract class OperatorsShouldHaveSymmetricalOverloadsAnalyzer : Diagnost
 
```

Comment:
Do not forget to add the Attribute `<DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)>` to the base type